From ed18d718743ad6547ace9bb80a710eb4b1d590c4 Mon Sep 17 00:00:00 2001
From: Jack Cherng <jfcherng@gmail.com>
Date: Fri, 15 May 2020 21:41:40 +0800
Subject: [PATCH] chore: patch RDM WelcomeTab for repo information

Signed-off-by: Jack Cherng <jfcherng@gmail.com>
---
 src/qml/WelcomeTab.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/qml/WelcomeTab.qml b/src/qml/WelcomeTab.qml
index 0dd2f0bc..46ef9b91 100644
--- a/src/qml/WelcomeTab.qml
+++ b/src/qml/WelcomeTab.qml
@@ -27,7 +27,7 @@ BetterTab {
             ColumnLayout {
                 Layout.fillWidth: true
                 RichTextWithLinks { Layout.fillWidth: true; html: '<span style="font-size:40px;">Redis Desktop Manager</span>'}
-                RichTextWithLinks { Layout.fillWidth: true; html: '<span style="font-size: 13px;"><b>Version</b> ' + Qt.application.version +'</span>' }
+                RichTextWithLinks { Layout.fillWidth: true; html: '<span style="font-size: 13px;"><b>Version</b> <a href="https://github.com/jfcherng/RedisDesktopManager-Windows">' + Qt.application.version +'</a> @ <a href="https://github.com/jfcherng">jfcherng</a></span>' }
                 RichTextWithLinks { Layout.fillWidth: true;  html:  '<span style="font-size: 11px;">Powered by awesome <a href="https://github.com/uglide/RedisDesktopManager/tree/2019/3rdparty">open-source software</a>, '
                                                                     + '<br /><a href="http://icons8.com/">icons from icons8.com</a> and '                                                                    + '<a href="http://redis.io/">Redis Logo</a>.</span>'}
             }
-- 
2.26.2.windows.1

