From 916c58cf45ce6925781a2fd826e9eb94ea9b7d59 Mon Sep 17 00:00:00 2001
From: Jack Cherng <jfcherng@gmail.com>
Date: Wed, 21 Oct 2020 19:53:06 +0800
Subject: [PATCH] chore: patch RDM WelcomeTab for repo information

Signed-off-by: Jack Cherng <jfcherng@gmail.com>
---
 src/qml/WelcomeTab.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/qml/WelcomeTab.qml b/src/qml/WelcomeTab.qml
index 199b9742..be01db79 100644
--- a/src/qml/WelcomeTab.qml
+++ b/src/qml/WelcomeTab.qml
@@ -27,7 +27,7 @@ BetterTab {
             ColumnLayout {
                 Layout.fillWidth: true
                 RichTextWithLinks { Layout.fillWidth: true; html: '<span style="font-size:30px;">RDM.dev - GUI for Redis<sup>®</sup></span>'}
-                RichTextWithLinks { Layout.fillWidth: true; html: '<span style="font-size: 13px;"><b>Version</b> ' + Qt.application.version +'</span>' }
+                RichTextWithLinks { Layout.fillWidth: true; html: '<span style="font-size: 13px;"><b>Version</b> <a href="https://github.com/jfcherng/RedisDesktopManager-Windows">' + Qt.application.version +'</a> @ <a href="https://github.com/jfcherng">jfcherng</a></span>' }
                 RichTextWithLinks { Layout.fillWidth: true;  html:  '<span style="font-size: 11px;">Powered by awesome <a href="https://github.com/uglide/RedisDesktopManager/tree/2019/3rdparty">open-source software</a> and '
                                                                     + '<br /><a href="http://icons8.com/">icons from icons8.com</a>.</span>'}
             }
-- 
2.29.0.windows.1

