From 56c7d99441d728907e193ec6337ed1c842328393 Mon Sep 17 00:00:00 2001
From: Jack Cherng <jfcherng@gmail.com>
Date: Mon, 20 Apr 2020 17:11:24 +0800
Subject: [PATCH] chore: patch RDM WelcomeTab for repo information

Signed-off-by: Jack Cherng <jfcherng@gmail.com>
---
 src/qml/WelcomeTab.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/qml/WelcomeTab.qml b/src/qml/WelcomeTab.qml
index 7dd949bf..61d1dd42 100644
--- a/src/qml/WelcomeTab.qml
+++ b/src/qml/WelcomeTab.qml
@@ -26,7 +26,7 @@ BetterTab {
             ColumnLayout {
                 Layout.fillWidth: true
                 RichTextWithLinks { Layout.fillWidth: true; html: '<span style="font-size:32pt;">Redis Desktop Manager</span>'}
-                RichTextWithLinks { Layout.fillWidth: true; html: '<span style="font-size: 13px;"><b>Version</b> ' + Qt.application.version +'</span>' }
+                RichTextWithLinks { Layout.fillWidth: true; html: '<span style="font-size: 13px;"><b>Version</b> <a href="https://github.com/jfcherng/RedisDesktopManager-Windows">' + Qt.application.version +'</a> @ <a href="https://github.com/jfcherng">jfcherng</a></span>' }
                 RichTextWithLinks { Layout.fillWidth: true;  html:  '<span style="font-size: 11px;">Powered by awesome <a href="https://github.com/uglide/RedisDesktopManager/tree/2019/3rdparty">open-source software</a>, '
                                                                     + '<br /><a href="http://icons8.com/">icons from icons8.com</a> and '
                                                                     + '<a href="http://redis.io/">Redis Logo</a>.</span>'}
-- 
2.26.1.windows.1

