From 587f69eacf04526990496bd3e740489c7ab00dc2 Mon Sep 17 00:00:00 2001
From: Jack Cherng <jfcherng@gmail.com>
Date: Mon, 29 Mar 2021 16:04:58 +0800
Subject: [PATCH] chore: do not install system-wide vcredist

Signed-off-by: Jack Cherng <jfcherng@gmail.com>
---
 build/windows/installer/installer.nsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/windows/installer/installer.nsi b/build/windows/installer/installer.nsi
index d49dd548..d2a117bc 100644
--- a/build/windows/installer/installer.nsi
+++ b/build/windows/installer/installer.nsi
@@ -85,7 +85,7 @@ Section -Main SEC0000
     SetOutPath $INSTDIR    
     File /r resources\*
     WriteRegStr HKLM "${REGKEY}\Components" Main 1
-    !insertmacro InstallVCredist
+    # !insertmacro InstallVCredist
     BringToFront
 SectionEnd
 
-- 
2.31.1.windows.1

