From eda4c3752f29772132291a327e747fc489b87fa4 Mon Sep 17 00:00:00 2001
From: Jack Cherng <jfcherng@gmail.com>
Date: Mon, 29 Mar 2021 16:02:15 +0800
Subject: [PATCH] chore: allow specifying detailed version info in the NSI

Signed-off-by: Jack Cherng <jfcherng@gmail.com>
---
 build/windows/installer/installer.nsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/build/windows/installer/installer.nsi b/build/windows/installer/installer.nsi
index d2a117bc..a1fee208 100644
--- a/build/windows/installer/installer.nsi
+++ b/build/windows/installer/installer.nsi
@@ -48,17 +48,17 @@ Var StartMenuGroup
 !insertmacro MUI_LANGUAGE English
 
 # Installer attributes
-OutFile rdm-${VERSION}.exe
+OutFile rdm-${RELEASE_TAG}.exe
 InstallDir $PROGRAMFILES64\RDM
 CRCCheck on
 XPStyle on
 ShowInstDetails show
 VIProductVersion ${VERSION}.0
 VIAddVersionKey /LANG=${LANG_ENGLISH} ProductName "RDM"
-VIAddVersionKey /LANG=${LANG_ENGLISH} ProductVersion "${VERSION}"
+VIAddVersionKey /LANG=${LANG_ENGLISH} ProductVersion "${RELEASE_TAG}"
 VIAddVersionKey /LANG=${LANG_ENGLISH} CompanyName "${COMPANY}"
 VIAddVersionKey /LANG=${LANG_ENGLISH} CompanyWebsite "${URL}"
-VIAddVersionKey /LANG=${LANG_ENGLISH} FileVersion "${VERSION}"
+VIAddVersionKey /LANG=${LANG_ENGLISH} FileVersion "${RELEASE_TAG}"
 VIAddVersionKey /LANG=${LANG_ENGLISH} FileDescription ""
 VIAddVersionKey /LANG=${LANG_ENGLISH} LegalCopyright ""
 InstallDirRegKey HKLM "${REGKEY}" Path
@@ -104,7 +104,7 @@ Section -post SEC0001
     CreateShortcut "$SMPROGRAMS\$StartMenuGroup\$(^UninstallLink).lnk" $INSTDIR\uninstall.exe
 
     WriteRegStr HKLM "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" DisplayName "$(^Name)"
-    WriteRegStr HKLM "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" DisplayVersion "${VERSION}"
+    WriteRegStr HKLM "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" DisplayVersion "${RELEASE_TAG}"
     WriteRegStr HKLM "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" Publisher "${COMPANY}"
     WriteRegStr HKLM "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" URLInfoAbout "${URL}"
     WriteRegStr HKLM "SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\$(^Name)" DisplayIcon $INSTDIR\uninstall.exe
-- 
2.31.1.windows.1

