From be1b89ded69fbdb9630bb64131936abc42ec35e0 Mon Sep 17 00:00:00 2001
From: Jack Cherng <jfcherng@gmail.com>
Date: Fri, 11 Sep 2020 16:00:05 +0800
Subject: [PATCH] [PATCH] chore: use Python 3.8 binding

Signed-off-by: Jack Cherng <jfcherng@gmail.com>
---
 3rdparty/pyotherside.pri | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/3rdparty/pyotherside.pri b/3rdparty/pyotherside.pri
index a9b64ddf..78fbca97 100644
--- a/3rdparty/pyotherside.pri
+++ b/3rdparty/pyotherside.pri
@@ -1,17 +1,17 @@
 
 # Python
 win32* {
-    QMAKE_LIBS += -LC:\Python37-x64\libs -lpython37
-    INCLUDEPATH += C:\Python37-x64\include\
+    QMAKE_LIBS += -LC:\Python38-x64\libs -lpython38
+    INCLUDEPATH += C:\Python38-x64\include\
 } else {
     unix:macx {
       exists($$PWD/python-3) {
         message("Using Python from 3rdparty dir")
-        LIBS += $$PWD/python-3/lib/libpython3.7m.dylib
-        INCLUDEPATH += $$PWD/python-3/include/python3.7m
+        LIBS += $$PWD/python-3/lib/libpython3.8m.dylib
+        INCLUDEPATH += $$PWD/python-3/include/python3.8m
 
         #deployment
-        PY_DATA_FILES.files = $$PWD/python-3/lib/libpython3.7m.dylib
+        PY_DATA_FILES.files = $$PWD/python-3/lib/libpython3.8m.dylib
         PY_DATA_FILES.path = Contents/Frameworks
         QMAKE_BUNDLE_DATA += PY_DATA_FILES
 
-- 
2.28.0.windows.1

