From 7050b33695e7cd2d4c6e142aaea2bcce1ee36393 Mon Sep 17 00:00:00 2001
From: Jack Cherng <jfcherng@gmail.com>
Date: Sat, 5 Jun 2021 07:11:17 +0800
Subject: [PATCH] feat: use Python 3.8 (supports Windows 7)

Signed-off-by: Jack Cherng <jfcherng@gmail.com>
---
 3rdparty/pyotherside.pri | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/3rdparty/pyotherside.pri b/3rdparty/pyotherside.pri
index 8e91add2..7704d402 100644
--- a/3rdparty/pyotherside.pri
+++ b/3rdparty/pyotherside.pri
@@ -1,7 +1,7 @@
 
 # Python
-PY_VERSION="39"
-PY_LIB_SUFFIX="3.9"
+PY_VERSION="38"
+PY_LIB_SUFFIX="3.8"
 
 win32* {
     QMAKE_LIBS += -LC:\Python$${PY_VERSION}-x64\libs -lpython$${PY_VERSION}
-- 
2.31.1.windows.1

