From 756330d99499226e79e3643a3f1e317017f41f83 Mon Sep 17 00:00:00 2001
From: Jack Cherng <jfcherng@gmail.com>
Date: Mon, 20 Apr 2020 17:08:22 +0800
Subject: [PATCH] fix: SSH header file path

Signed-off-by: Jack Cherng <jfcherng@gmail.com>
---
 src/qredisclient/connection.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/qredisclient/connection.cpp b/src/qredisclient/connection.cpp
index 17c2036..321968a 100644
--- a/src/qredisclient/connection.cpp
+++ b/src/qredisclient/connection.cpp
@@ -12,7 +12,7 @@
 #include "utils/sync.h"
 
 #ifdef SSH_SUPPORT
-#include "sshtransporter.h"
+#include "transporters/sshtransporter.h"
 #endif
 
 inline void initResources() { Q_INIT_RESOURCE(lua); }
-- 
2.26.1.windows.1

